hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsOperationConstants.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.hadoop.fs.azurebfs.constants;
+
+public final class AbfsOperationConstants {
+    public static final String ACCESS = "AS";
```

Comment:
these are actually HdfsOperationConstants right ? Not AbfsOperation. Class needs rename.

Reply:
renamed to HdfsOperationConstants